api/src/pcapi/tasks/__init__.py

```diff
@@ -3,6 +3,8 @@
 
 def install_handlers(app: Flask) -> None:
     # pylint: disable=unused-import
+    from pcapi.core.providers import tasks
```

Comment:
(détail) Je propose d'importer différemment, comme suit : 

```suggestion
    import pcapi.core.providers.tasks
```

Si on écrit `from ... import tasks`, pylint nous lèvera un avertissement "reimported" quand on voudra transformer les autres imports de la même façon :  

```python
     from pcapi.core.providers import tasks
     from pcapi.foo.batch import tasks
     from pcapi.foo.sendinblue import tasks
```